common/lib/xmodule/xmodule/course_module.py

```diff
@@ -293,15 +293,10 @@ class CourseFields(object):
             '{"id": "i4x-InstitutionName-CourseNumber-course-CourseRun"}. For example, one discussion '
             'category may be "Lydian Mode": {"id": "i4x-UniversityX-MUS101-course-2015_T1"}. The "id" '
             'value for each category must be unique. In "id" values, the only special characters that are '
-            'supported are underscore, hyphen, and period.'
-        ),
-        scope=Scope.settings
-    )
-    discussion_default_topic_id = String(
-        display_name=_("Discussion Default Topic ID"),
-        help=_(
-            "Enter the id of a discussion category to be used by default when adding posts from "
-            "the Discussion page."
+            'supported are underscore, hyphen, and period. You can also specify a CategoryName as the default '
+            'for new posts in the Discussion page by setting its "default" attribute to "true". For example, '
+            '"Lydian Mode": {"id": "i4x-UniversityX-MUS101-course-2015_T1", "default": "true"}.'
```

Comment:
CC @cahrens in case you have a preference here since I see this was quoted in your example.